app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = t(".title")
```

Comment:
dùng ở trong helper hoặc model thì phải khai báo rõ syntax nó mới nhận nha em
```suggestion
    base_title = I81n.t(".title")
```